app/views/va_facilities/_index_table_row.html.erb

```diff
@@ -4,7 +4,7 @@
     <td class="grid-col-1"><%= facility.street_address_state %></td>
     <td class="grid-col-1"><a class="usa-link" href='/visns/<%= facility.visn.number %>'><%= facility.visn.number %></a></td>
     <td class="grid-col-2"><%= facility.fy17_parent_station_complexity_level.match(/^1/) ? facility.fy17_parent_station_complexity_level[0..1].upcase : facility.fy17_parent_station_complexity_level[0..0].upcase %></td>
-    <td class="grid-col-2"><%= facility.practices_created_count %></td>
-    <td class="grid-col-2"><%= facility.practices_adopted_count  %></td>
+    <td class="grid-col-2"><%= created_practices_count(facility.id) %></td>
+    <td class="grid-col-2"><%= adopted_practices_count(facility.id)  %></td>
```

Comment:
We can just call the instance variable here

Reply:
The instance var will need to be updated for each row... so we have to call the methods here... we can't just reference the instance var.

Reply:
If that's the case, can we move these methods to a `va_facilities_helper.rb` file? That way we won't need the `helper_method` calls at the top of the class and we can call them in the view whenever we need to.

Reply:
Yep that works.  Thx Josh